src/main/java/com/google/devtools/build/lib/bazel/bzlmod/BazelDepGraphFunction.java

```diff
@@ -249,6 +264,27 @@ static BzlmodFlagsAndEnvVars getFlagsAndEnvVars(Environment env) throws Interrup
     return extensionUsagesTableBuilder.buildOrThrow();
   }
 
+  private static ImmutableBiMap<RepositoryName, ModuleKey> computeCanonicalRepoNameLookup(
+      ImmutableMap<ModuleKey, Module> depGraph) {
+    // Find those modules of which there is only a single version in the dep graph. Currently, the
+    // only way to have multiple versions of a module in the dep graph is via
+    // multiple_version_override.
+    ImmutableSet<String> uniqueVersionModules =
+        depGraph.keySet().stream()
+            .collect(groupingBy(ModuleKey::getName, counting()))
```

Comment:
could pass in the overrides and filter out multiple_version_overrides instead, WDYT? I think that's slightly more intuitive (and realisitically I don't think we'll ever introduce other ways to have multiple versions of a module)

Reply:
`multiple_version_override` doesn't allow less than 2 versions. But anyway, I don't feel too strongly about this, and there's pros and cons either way, so I'm fine with keeping it as is.